lms/static/js/courseware/accordion_events.js

```diff
@@ -0,0 +1,20 @@
+;(function(define) {
+    'use strict';
+
+    define(['jquery', 'logger'], function ($, Logger) {
+        return function () {
+            $(".accordion-nav").click(function(event) {
+                Logger.log(
+                    "edx.ui.lms.outline.selected",
+                    {
+                        name: "edx.ui.lms.outline.selected",
+                        event_type: "edx.ui.lms.outline.selected",
+                        current_url: window.location.href,
+                        target_url: event.currentTarget.href,
+                        target_name: $(this).find("p.accordion-display-name").text(),
```

Comment:
Nit: Why precede the query with a "p" and not just do a ".accordion-display-name"?